README.md

```diff
@@ -7,6 +7,7 @@ https://discordjs.guide/
 ### Installation
 1. Clone repository into a directory
 2. `cd` into directory and run `npm install`.
+3. `npm run setup.js` to run initial setup (create config.js)
```

Comment:
Oops. This is because I removed postinstall to run setup.js to unblock the CI tests from running properly.

Reply:
haha yea I ran into minor issues with this when setting up on my other computer